src/components/document/sort-work-document-area.tsx

```diff
@@ -28,8 +28,6 @@ export const SortWorkDocumentArea: React.FC<IProps> = observer(function SortWork
   const openDocument = store.documents.getDocument(openDocumentKey) ||
                        store.networkDocuments.getDocument(openDocumentKey);
   const isVisible = openDocument?.isAccessibleToUser(user, store.documents);
-  const showPlayback = user.type && !openDocument?.isPublished
-                       ? appConfigStore.enableHistoryRoles.includes(user.type) : false;
```

Comment:
@lbondaryk specified that this should just be visible in all cases, but @scytacki and @kswenson I wasn't sure if programmatically there were cases we should not let fall through, and/or config that should still stay applicable here?

E.g. should I preserve at least this condition:

```ts
const showPlayback = user.type && !openDocument?.isPublished;
``` 
and/or something that checks the role config?